### PR TITLE
simplify miniconda recommendations

### LINKED ISSUE #571: Remove text about miniforge/conda-forge not having good GPU support
This is not true anymore.

### PR DESCRIPTION
**Pull request by mattip**
#### Brief description of what is fixed or changed

Fixes #571 by removing the sections about linux. Conda-forge now has gpu packages. Since the changes were all deletions, I took the liberty of modifying the translations as well, I hope it is appropriate. Along the way I made an opinionated edit to removed the recommendation to use the system libraries on linux since the versions lag way behind what is available on conda/PyPI.

### CODE REVIEW

File: content/en/install.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ On all of Windows, macOS, and Linux:
 
 ### Advanced users
 
-#### Windows or macOS
+#### Conda
 
 - Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
```
Comment (rgommers) [inline content/en/install.md:81]: @mattip can you change this to Miniforge then? And the drop the bit below about `defaults`? We can and should recommend `conda-forge` without any disclaimer now (and that was the point of the issue you're addressing)

Comment (mattip) [reply content/en/install.md]: Done for English. Is there someone to ping for the translations?

Comment (rgommers) [reply content/en/install.md]: No, they show up (with notifications) automatically in Crowdin if you don't touch them. This is just a name, though, so why not just make the change in the translations?

Comment (mattip) [reply content/en/install.md]: > And the drop the bit below about defaults?

This is the part that requires help with translation

Comment (rgommers) [reply content/en/install.md]: Okay then let's leave that as is. It'll show up in Crowdin.

File: content/en/install.md [hunk 2]
```diff
@@ -76,38 +76,14 @@ On all of Windows, macOS, and Linux:
...
-  GPU packages yet).
-
-Otherwise:
 - Install [Miniforge](https://github.com/conda-forge/miniforge).
 - Keep the `base` conda environment minimal, and use one or more
   [conda environments](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
   to install the package you need for the task or project you're working on.
-
+- Make `conda-forge` your default channel by [setting the channel
+  priority](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge).
```
Comment (rgommers) [inline content/en/install.md:86]: This can be removed as I tried to say in my previous comment. It is already the default in Miniforge.

Comment (mattip) [reply content/en/install.md:86]: OK. The comment
> We can and should recommend conda-forge without any disclaimer now

confused me. Removing entirely.

**Review by rgommers — APPROVED** [review]
LGTM, thanks @mattip

**Pull request merged**